css/amp.css

```diff
@@ -167,6 +167,24 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+.i-amphtml-blur-loading .i-amphtml-fill-content {
+  opacity: 0 !important;
+}
+
+.i-amphtml-blur-loaded .i-amphtml-fill-content {
+  opacity: 1 !important;
+  transition: opacity 1s cubic-bezier(0.33, 0.33, 1, 1) !important;
```

Comment:
How did you decide on these cubic-bezier easing values?

Reply:
Talked to about this, but for other reviewers reading this, we decided to start out by removing the cubic-bezier for now since it was linear anyways.